src/backfill_unpublished.py

```diff
@@ -0,0 +1,28 @@
+import adaptor, fs_adaptor, utils
+import json
+
+# not working, untested
+
+def main():
+    cmd = [
+        adaptor.find_lax(), # /srv/lax/manage.sh
+        "status",
+        "article-versions.invalid-unpublished.list"
+    ]
+    lax_stdout = None
+    rc, lax_stdout = utils.run_script(cmd)
+
+    requests = []
+    for av in json.loads(lax_stdout)['article-versions.invalid-unpublished.list']:
```

Comment:
I think this is what you're after: https://github.com/elifesciences/lax/pull/99/files/2f41354d08dc436910c4c94bf59973781e78da77#diff-88497525781680406c91290667cfbce3R51

that `article-versions.invalid-unpublished.list` is just a path into a dict, in this case `{'article-versions': {'invalid-unpublished': 'list': [...]}}` that is dealt with here: https://github.com/elifesciences/lax/pull/99/files/2f41354d08dc436910c4c94bf59973781e78da77#diff-4af08ff4de97e518a6cc95efb2fe826cR28

you could do the same with `jq`, but I don't like `jq` except for really basic things.